lib/renderer.js

```diff
@@ -1271,13 +1272,16 @@
             hasComplexVectorMask = layer && layer.path &&
                 layer.path.pathComponents && Array.isArray(layer.path.pathComponents) &&
                 (layer.path.pathComponents.length > 1),
+            nonIntegralBounds = (layer.bounds.top % 1 !== 0) || (layer.bounds.bottom % 1 !== 0) ||
```

Comment:
1. This needs to be made safe for non-layer situations such as Layer Comps.
`Render failed: Layer Comp 1.png [TypeError: Cannot read property 'bounds' of undefined]`
1. Consider using `Number.isInteger()` for readability